riverdriver/river_driver_interface.go

```diff
@@ -453,7 +479,7 @@ type JobSetStateIfRunningManyParams struct {
 	MetadataDoMerge []bool
 	MetadataUpdates [][]byte
 	ScheduledAt     []*time.Time
-	Schema          string
+	Schema          string // TODO(brandur): Try to push this up a layer; it's awkward being the only non-slice on here.
```

Comment:
Hmm, while it's maybe a little awkward, it would also seem weird to do anything else here given there can only be one value no matter how may rows are updated (which is unique from the other fields).

Reply:
Yeah, true. I took that TODO back out. This seems fine for now.